gooddata-sdk/gooddata_sdk/catalog/export/request.py

```diff
@@ -0,0 +1,31 @@
+# (C) 2023 GoodData Corporation
+from typing import Any, Dict, Optional, Type
+
+import attr
+
+from gooddata_api_client.model.tabular_export_request import TabularExportRequest
+from gooddata_sdk.catalog.base import Base
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class ExportRequest(Base):
+    format: str
+    execution_result: str
```

Comment:
Please, rename it to `execution_result_id`, so it is more obvious what is the purpose of this property

Reply:
I think that it would be more appropriate, but this class mirrors the `TabularExportRequest` class which is generated by the generator and therefore I cannot have a custom name because it would not match the current implementation.